### PR TITLE
fix migrations

### PR DESCRIPTION
**Pull request by internetti**

### CODE REVIEW

File: internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ALTER TABLE countries
+    -- drop jwt_group column
+    DROP COLUMN jwt_group;
```
Comment (neb42) [inline internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:3]: `IF EXISTS`

File: internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+ALTER TABLE countries
+    -- add nrc_organisation column
+    ADD COLUMN nrc_organisation      VARCHAR(255)         NOT NULL DEFAULT '';
```
Comment (neb42) [inline internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:6]: `IF NOT EXISTS`

File: internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql (hunk 1 above)
Comment (ludydoo) [reply internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:3]: > `IF EXISTS`

I don't think this command exists for columns

File: internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql (hunk 2 above)
Comment (ludydoo) [reply internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:6]: Same comment as above

Comment (ludydoo) [reply internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:6]: Maybe I'm wrong

Comment (ludydoo) [reply internal/db/migrations/postgres/018_add_nrc_organisation_to_country.up.sql:6]: I'm wrong.

**Review by ludydoo — APPROVED** [review]

**Pull request merged**